.gitignore

```diff
@@ -14,6 +14,9 @@ Icon
 # Thumbnails
 ._*
 
+# IntelliJ IDEA
+.idea
+
```

Comment:
Please remove this change, the right way to handle IDE-specific ignore files like this is to use a global ignore file that applies to your whole computer: https://help.github.com/en/github/using-git/ignoring-files#create-a-global-gitignore